spec/models/ext_management_system_spec.rb

```diff
@@ -25,6 +25,7 @@
       scvmm
       vmwarews
       azure
+      ansible_tower_configuration
```

Comment:
Ansible Tower only has a Configuration manager at this time.  I added configuration to be consistent with foreman, although I think it should really be `ansible_tower_configuration_manager` or `configuration_manager_ansible_tower`.